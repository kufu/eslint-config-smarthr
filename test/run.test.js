const runLint = require('./lib/runLint')

describe('fixtures', () => {
  it('should match the snapshot', async () => {
    const result = await runLint('./index.js', './test/fixtures')
    expect(result).toMatchSnapshot()
  })
})
