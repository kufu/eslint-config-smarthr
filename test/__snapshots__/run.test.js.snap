// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fixtures should match the snapshot 1`] = `
Object {
  "Counter.tsx": Object {
    "errors": Array [
      "prettier/prettier",
      "prettier/prettier",
      "jsx-quotes",
    ],
    "warnings": Array [
      "react/no-children-prop",
      "jsx-a11y/aria-role",
    ],
  },
  "mod.ts": Object {
    "errors": Array [],
    "warnings": Array [],
  },
  "sum.ts": Object {
    "errors": Array [
      "default-param-last",
      "prefer-regex-literals",
      "no-shadow",
      "prettier/prettier",
    ],
    "warnings": Array [
      "no-div-regex",
    ],
  },
}
`;
